open-machine-learning-jupyter-book/STYLE_GUIDE.md

```diff
@@ -210,6 +210,23 @@ assets/
 
 Note that we may need to set the `width` to `105%` so that all content of `index.html` will be rendered correctly. Also, the `height` has to be set manually.
 
+## Visualization parts
+
+1\. Get the target html file, make sure all the bundled files are downloaded and the html file can be run separately locally.
+
+2\. Keep only the visualization content, no additional description is needed.
+
+3\. add the html file and bundle to the `open-machine-learning-jupyter-book\assets\html` counterpart.
+
+4\. Embed html within the corresponding section. html embedding method refer to ipython. For example:
```

Comment:
Could you use grammarly or any other tool to fix the syntax error? Also, you can use got or Claud to improve the writing.